starboard/linux/x64x11/clang/3.9/BUILD.gn

```diff
@@ -25,6 +25,9 @@ static_library("starboard_platform") {
     "//starboard/shared/starboard/player:video_dmp",
   ]
 
+  # This is needed at link time, but not directly linked in
+  data_deps =
```

Comment:
Deps should come after configs https://chromium.googlesource.com/dart/dartium/src/+/HEAD/tools/gn/docs/style_guide.md#Ordering-within-a-target

Reply:
Moved